### PR TITLE
chore: update CHANGELOG for v0.20.0

### PR DESCRIPTION
**Pull request by shaneutt**
release `v0.20.0`, needed soon by KIC for the relevant workspace fixes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,24 @@
...
 
+## [v0.20.0] - 2021/07/07
+
+### Added
+
+- FriendlyName is now defined for entities based on deck types.
+  [#68](https://github.com/Kong/go-kong/pull/68)
+- Information Service is now present in the top of the Root endpoint for Kong.
+  [#65](https://github.com/Kong/go-kong/pull/65)
+- Client now properly handles Kong workspaces.
```
Comment (rainest) [inline CHANGELOG.md:35]:
```suggestion
- Implemented client-level support for Kong workspaces.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,24 @@
 - [0.2.0](#020---20181219)
 - [0.1.0](#010---20181201)
 
+## [v0.20.0] - 2021/07/07
+
+### Added
+
+- FriendlyName is now defined for entities based on deck types.
+  [#68](https://github.com/Kong/go-kong/pull/68)
+- Information Service is now present in the top of the Root endpoint for Kong.
```
Comment (rainest) [inline CHANGELOG.md:33]:
```suggestion
- Added Info service for interacting with information exposed by the admin API root endpoint.
```

**Review by rainest — CHANGES_REQUESTED** [review]
TOC needs this in the list:

```
- [v0.20.0](#v0200---202100707)
```

End needs the link to the diff:

```
[v0.20.0]: https://github.com/Kong/go-kong/compare/v0.19.0...v0.20.0
```

**Comment by shaneutt** [comment]
> TOC needs this in the list:
> 
> ```
> - [v0.20.0](#v0200---202100707)
> ```
> 
> End needs the link to the diff:
> 
> ```
> [v0.20.0]: https://github.com/Kong/go-kong/compare/v0.19.0...v0.20.0
> ```

whoops, thanks. fixed 723ecb17ac66559b16378c3769e0af11e13907c7

**Review by rainest — APPROVED** [review]

**Pull request merged**